webpack.config.js

```diff
@@ -16,6 +16,7 @@ const config = [{
     './modules/issue_tracker/js/index.js': './modules/issue_tracker/jsx/index.js',
     './modules/candidate_parameters/js/CandidateParameters.js': './modules/candidate_parameters/jsx/CandidateParameters.js',
     './modules/configuration/js/SubprojectRelations.js': './modules/configuration/jsx/SubprojectRelations.js',
+    './modules/create_timepoint/js/create_timepoint.js': './modules/create_timepoint/jsx/create_timepoint.js',
```

Comment:
i suggest renaming the file to `createTimepointIndex.js` for no reason other than consistency :)

Reply:
Yah I saw the other day we have a template for react that @HenriRabalais sent me in another PR. Thanks @zaliqarosli I'll change it.